docs/src/tutorials/conic/min_ellipse.jl

```diff
@@ -3,48 +3,102 @@
 # obtain one at https://mozilla.org/MPL/2.0/.                                   #src
 
-# # Minimum ellipses
-
-# This example is from the Boyd & Vandenberghe book "Convex Optimization". Given
-# a set of ellipses centered on the origin
-#
-#     E(A) = { u | u^T inv(A) u <= 1 }
-#
-# find a "minimal" ellipse that contains the provided ellipses.
-#
-# We can formulate this as an SDP:
-#
-#     minimize    trace(WX)
-#     subject to  X >= A_i,    i = 1,...,m
-#                 X PSD
-#
-# where W is a PD matrix of weights to choose between different solutions.
+# # Minimal ellipses
+
+# This example comes from section 8.4.1 of the book *Convex Optimization* (Boyd and Vandenberghe, 2004).
+
+# Given a set of `m` ellipses `E(A, b, c) = { x : x' A x + 2 b' x + c ≤ 0 }`, we find the ellipse of smallest area that encloses the given ellipses.
+
+# It is convenient to parameterize as the ellipse `{ x : || Px + q || ≤ 1 }`.
+
+# Then the optimal `P` and `q` are given by the convex program
+
+# ````
+#       maximize   log(det(P))
+      
+#     subject to   t[i] ≥ 0, i = 1 ... m
+    
+#                  [ P^2 - t[i] A[i]   P q - t[i] b[i]       0
+#                    P q - t[i] b[i]    -1 - t[i] c[i]    (P q)'
+#                                 0              (P q)   - P^2   ]    PSD, i = 1 ... m
+# ````
+# with helper variables `t[i]`. 
 
 using JuMP
-import LinearAlgebra
-import SCS
-import Test
-
-function example_min_ellipse()
-    ## We will use three ellipses: two "simple" ones, and a random one.
-    As = [
-        [2.0 0.0; 0.0 1.0],
-        [1.0 0.0; 0.0 3.0],
-        [2.86715 1.60645; 1.60645 1.12639],
+using SCS
+using Plots
+
+# Input ellipses, parameterized as
+# x' A x + 2 b' x + c ≤ 0
+As = [
+        [1.2576 -0.3873; -0.3873 0.3467],
+        [1.4125 -2.1777; -2.1777 6.7775],
+        [1.7018  0.8141;  0.8141 1.7538],
+        [0.9742 -0.7202; -0.7202 1.5444],
+        [0.6798 -0.1424; -0.1424 0.6871],
+        [0.1796 -0.1423; -0.1423 2.6181]
+    ]
+bs = [
+        [ 0.2722,  0.1969],
+        [-1.228 , -0.0521],
+        [-0.4049,  1.5713],
+        [ 0.0265,  0.5623],
+        [-0.4301, -1.0157],
+        [-0.3286,  0.557 ]
     ]
-    ## We change the weights to see different solutions, if they exist
-    weights = [1.0 0.0; 0.0 1.0]
-    model = Model(SCS.Optimizer)
-    set_silent(model)
-    @variable(model, X[i = 1:2, j = 1:2], PSD)
-    @objective(model, Min, LinearAlgebra.tr(weights * X))
-    @constraint(model, [As_i in As], X >= As_i, PSDCone())
-    optimize!(model)
-    Test.@test termination_status(model) == OPTIMAL
-    Test.@test primal_status(model) == FEASIBLE_POINT
-    Test.@test objective_value(model) ≈ 6.46236 atol = 1e-4
-    Test.@test value.(X) ≈ [3.1651 0.8022; 0.8022 3.2972] atol = 1e-4
-    return
+cs = [0.1831, 0.3295, 0.2077, 0.2362, 0.3284, 0.4931]
+
+function minimal_ellipse(As, bs, cs)
+    # Build the model under the change of variables
```

Comment:
```suggestion
    ## Build the model under the change of variables
```

To make a comment inside a Julia block, you need double hash signs: `##`